<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerTableWindow</class>
 <widget class="QWidget" name="ManagerTableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>641</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="projectTab">
    <attribute name="title">
     <string>  项目表  </string>
    </attribute>
    <widget class="QTableWidget" name="projectTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1021</width>
       <height>601</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>20</number>
     </property>
     <property name="columnCount">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </widget>
   <widget class="QWidget" name="userTab">
    <attribute name="title">
     <string>  用户表  </string>
    </attribute>
    <widget class="QTableWidget" name="userTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1021</width>
       <height>601</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </widget>
   <widget class="QWidget" name="deviceTab">
    <attribute name="title">
     <string>  设备表  </string>
    </attribute>
    <widget class="QTableWidget" name="deviceTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1021</width>
       <height>601</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>13</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="newPushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePushButton">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>680</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>680</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchPushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>680</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>条件查找</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
